src/Tizen.NUI/src/internal/WebView/WebNewWindowPolicyDecisionMaker.cs

```diff
@@ -0,0 +1,246 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using System;
+using System.ComponentModel;
+
+namespace Tizen.NUI
+{
+    /// <summary>
+    /// It is a class for new window policy decision maker of web view.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class WebNewWindowPolicyDecisionMaker : Disposable
+    {
+        internal WebNewWindowPolicyDecisionMaker(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr, cMemoryOwn)
+        {
+        }
+
+        /// <summary>
+        /// Decision type
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public enum DecisionType
+        {
+            /// <summary>
+            /// Use
```

Comment:
What is `Use` in terms of the new window policy?

Reply:
Done.